src/robotide/ui/mainframe.py

```diff
@@ -593,6 +592,37 @@ def CreateSizeReportCtrl(self, width=80, height=80):
         ctrl = SizeReportCtrl(self, -1, wx.DefaultPosition, wx.Size(width, height), self._mgr)
         return ctrl
 
+    def show_confirm_reload_dlg(self, event):
+        msg = ['Workspace modifications detected on the file system.',
+               'Do you want to reload the workspace?',
+               'Answering <No> will overwrite the changes on disk.']
+        if self._controller.is_dirty():
+            msg.insert(2, 'Answering <Yes> will discard unsaved changes.')
+        ret = wx.MessageBox('\n'.join(msg), 'Files Changed On Disk',
+                            style=wx.YES_NO | wx.ICON_WARNING)
+        confirmed = ret == wx.YES
+        if confirmed:
+            # workspace_path should update after open dictionary/suite
+            # There're two scenarios:
+            # 1. path is a dictionary
```

Comment:
You mean "directory" not "dictionary" ;)

Reply:
yes, my typo.